test/pages/LifeEvent.spec.js

```diff
@@ -271,16 +271,19 @@ describe("Life Event page", () => {
     })
```

Comment:
How does this file relate to `test/Components/Accordion.spec.js`? (If at all?)

Reply:
Accordion is child component.
:key="`acc-key-${benefit.slug}`"
After adding slug, has key:
acc-key-slugTwo
acc-key-slugOne
acc-key-slugThree